### PR TITLE
Added all the html UI elements

### PR DESCRIPTION
**Pull request by Bagaric**
Added HTML elements for all the views along with their dependencies. Didn't implement state changes yet.  Also smaller webpack config changes. 
Let me know if we need to wait for any other skeleton changes before merging. I can adjust the PR accordingly.

### CODE REVIEW

File: app/components/ScheduleWizard/AwaitingMining/AwaitingMining.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
```
Comment (adibas03) [inline app/components/ScheduleWizard/AwaitingMining/AwaitingMining.js]: let's move this into the `components/ScheduleWizard` folder

File: app/components/ScheduleWizard/AwaitingMining/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export {default} from './AwaitingMining'
```
Comment (adibas03) [inline app/components/ScheduleWizard/AwaitingMining/index.js]: this file should be redundant.
The content should be moved into `components/ScheduleWizrd/index.js`

File: app/components/ScheduleWizard/BountySettings/BountySettings.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```
Comment (adibas03) [inline app/components/ScheduleWizard/BountySettings/BountySettings.js]: move this into the `components/ScheduleWizard` folder

File: app/components/ScheduleWizard/BountySettings/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export {default} from './BountySettings'
```
Comment (adibas03) [inline app/components/ScheduleWizard/BountySettings/index.js]: move contents into the move this into the `components/ScheduleWizard` index.js

File: app/components/ScheduleWizard/ConfirmSettings/ConfirmSettings.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
```
Comment (adibas03) [inline app/components/ScheduleWizard/ConfirmSettings/ConfirmSettings.js]: same as above

**Comment by Bagaric** [comment]
Resolving conflicts and implementing some @adibas03 suggestions.

File: app/components/ScheduleWizard/AwaitingMining/AwaitingMining.js (hunk 1 above)
Comment (Bagaric) [reply app/components/ScheduleWizard/AwaitingMining/AwaitingMining.js]: Done.

**Pull request merged**